.changeset/thin-dots-rest.md

```diff
@@ -0,0 +1,7 @@
+---
+'@srcbook/shared': minor
```

Comment:
I also just noticed this is `minor`, but for now I would like to keep all the versions on `patch`. Do you mind regenerating this changeset such that these are all `patch`?

Reply:
you don't need to regen just change the text to patch and it'll work